projects/telemetry/inc/nmea.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+// Computes the checksum for a given NMEA message.
+//
+// Starts at the second characer of `message`,
+// and continues until the `*` character, or until `message_len` characters
+// are processed, whichever comes first.
+//
+// `out` must be a pointer to at least 3 `char`s.
+void nmea_compute_checksum(char *message, size_t message_len, char *out);
```

Comment:
Rather than just telling the user through a comment, why not enforce it in your API? You can return a StatusCode.

Reply:
`void nmea_compute_checksum(char *message, size_t message_len, char *out, size_t out_len)` or something similar.